### PR TITLE
new metadata fields from Friday (see rocketchat-sar)

### PR DESCRIPTION
**Pull request by bbauerma**

### CODE REVIEW

**Review by AlenaDostalova — APPROVED** [review]
From my point of view can be merged. As Bernhard mentioned in TU chat, unknown type will probably not be needed.

File: src/medali/lib/plia/V01.ini [hunk 1]
```diff
@@ -65,4 +71,7 @@ run_number: pattern, R[0-9][0-9]
 satellite_id: list, S1A, S1B
 
 # acquisition mode id
-mode: list, IW, EW
\ No newline at end of file
+mode: list, IW, EW
+
+# orbit file of which category was used
+orbit_file_type: list, POEORB, RESORB, UnknownType, None
```
Comment (cnavacch) [inline src/medali/lib/plia/V01.ini]: If approved, please remove "UnknownType" and None. Only POEORB or RESORB should be allowed.

File: src/medali/lib/sig0/V01.ini [hunk 2]
```diff
@@ -83,4 +89,7 @@ polarisation: list, VV, VH, HH, HV
 satellite_id: list, S1A, S1B
 
 # acquisition mode id
-mode: list, IW, EW
\ No newline at end of file
+mode: list, IW, EW
+
+# orbit file of which category was used
+orbit_file_type: list, POEORB, RESORB, UnknownType, None
```
Comment (cnavacch) [inline src/medali/lib/sig0/V01.ini]: Same as for PLIA.

**Review by cnavacch — CHANGES_REQUESTED** [review]

File: src/medali/lib/sig0/V01.ini (hunk 2 above)
Comment (AlenaDostalova) [reply src/medali/lib/sig0/V01.ini]: removed now for both

**Pull request merged**